README.md

```diff
@@ -37,6 +37,7 @@ The goal of this first phase is to implement the above component (with low-level
 - [ ] make the existing test pass, add tests for edge cases
 - [ ] has no lintint errors (`yarn prettier && yarn lint && yarn typescript`)
 - [ ] has an ergonomic API
+- [ ] not using `<datalist>`
```

Comment:
Do you agree? I saw `ClickAwayListener` being used. I think `makeStyles` is allowed, not sure if we should list it since it's not a component.

```suggestion
- [ ] not using `<datalist>` or any component from MUI Core
```

Reply:
Sure, I would then broaden it to:
```suggestion
- [ ] not using `<datalist>` or any external React component
```